drivers/display/ssd1306_test.py

```diff
@@ -0,0 +1,72 @@
+# Driver test for SSD1306 display with SPI connection
+# MIT license; Copyright (c) 2017 Zoltan Gyarmati
+import machine
+
+width = 128
+height = 64
+
+def draw_testscreen(display):
+    try:
+        import pyb
+        rtc = pyb.RTC()
+    except:
+        rtc = None
+
+    # create M-logo
+    display.fill(0)
+    display.fill_rect(1, 1, 15, 15,1)
+    display.vline(4, 4, 12, 0)
+    display.vline(8, 1, 12, 0)
+    display.vline(12, 4, 12, 0)
+    display.vline(14, 13, 2, 0)
+    display.text('MicroPython',20,0)
+
+    display.hline(0,18,128,1)
```

Comment:
There seems to be an off-by-one error here. I think the coordinates are inclusive, i.e. it should be `display.hline(0, 18, 127, 1)` or `display.hline(0,18, width - 1,1)`.